File: src/main/java/com/p2p/lending/dao/LimitDao.java
package com.p2p.lending.dao;

import java.util.List;

import com.p2p.lending.pojo.Limi;

public interface LimitDao {

	//根据员工ID查询该用户权限的模块
	public List limitByeid(int eid);
	//删除
	public boolean limitdel(Integer eid);
	//添加
	public boolean limitadd(Limi limi);
	
	
}

File: src/main/java/com/p2p/lending/dao/LimitDao.kt
package com.p2p.lending.dao

import com.p2p.lending.entity.Limi

import org.springframework.stereotype.Repository

@Repository
interface LimitDao {

    //根据员工ID查询该用户权限的模块
    fun limitByeid(eid: Int): List<*>

    //删除
    fun limitdel(eid: Int?): Boolean

    //添加
    fun limitadd(limi: Limi): Boolean


}
